server/controllers/reviews.js

```diff
@@ -1,31 +1,33 @@
 const models = require('../models/reviews.js');
 
 module.exports = {
-  get(req, res) {
-    models.getReviews().then((allReviews) => {
-      // console.log('all messages from get are', allMessages);
+  getH(req, res) {
+    models.getReviews(req.params.hotelId).then((allReviews) => {
       res.status(200);
       res.send(allReviews);
     });
   },
+  getR(req, res) {
+    models.getOneReview(req.params.reviewId).then((review) => {
+      res.status(200);
+      res.send(review);
+    });
+  },
   post(req, res) {
-    // console.log('post req',req.body)
+    console.log('in post and body is ', req.body);
```

Comment:
Is this console log necessary?

Reply:
Yes, it is necessary for the functionality of my server